sass/_other.scss

```diff
@@ -142,50 +142,74 @@ body[class*='-moodle23'] .mod_turnitintooltwo  .centered_cb_cell input {
     border: 1px solid $light-grey;
 }
 
-.progress {
-    overflow: hidden;
-    height: 20px;
-    margin-bottom: 20px;
-    background-color: #f7f7f7;
-    background-image: -moz-linear-gradient(top,#f5f5f5,#f9f9f9);
-    background-image: -webkit-gradient(linear,0 0,0 100%,from(#f5f5f5),to(#f9f9f9));
-    background-image: -webkit-linear-gradient(top,#f5f5f5,#f9f9f9);
-    background-image: -o-linear-gradient(top,#f5f5f5,#f9f9f9);
-    background-image: linear-gradient(to bottom,#f5f5f5,#f9f9f9);
-    background-repeat: repeat-x;
-    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#fff5f5f5',endColorstr='#fff9f9f9',GradientType=0);
-    -webkit-box-shadow: inset 0 1px 2px rgba(0,0,0,.1);
-    -moz-box-shadow: inset 0 1px 2px rgba(0,0,0,.1);
-    box-shadow: inset 0 1px 2px rgba(0,0,0,.1);
-    -webkit-border-radius: 4px;
-    -moz-border-radius: 4px;
-    border-radius: 4px;
-}
-
-.progress .bar {
+#migration-progress-bar {
+    background-color: #FFFFFF;
+    height: 30px;
+    line-height: 28px;
+    border: 1px solid;
+    border-radius: 3px;
+    margin: 15px 0px 30px 0px;
+}
+#migration-progress {
+    background-color: #0073D2;
     width: 0%;
     height: 100%;
     color: #fff;
+    text-align: center;
     float: left;
-    font-size: 12px;
+    font-size: 14px;
+    text-align: center;
+}
+
+.enablemigrationtool {
     text-align: center;
-    text-shadow: 0 -1px 0 rgba(0,0,0,.25);
-    background-color: #0e90d2;
-    background-image: -moz-linear-gradient(top,#149bdf,#0480be);
-    background-image: -webkit-gradient(linear,0 0,0 100%,from(#149bdf),to(#0480be));
-    background-image: -webkit-linear-gradient(top,#149bdf,#0480be);
-    background-image: -o-linear-gradient(top,#149bdf,#0480be);
-    background-image: linear-gradient(to bottom,#149bdf,#0480be);
-    background-repeat: repeat-x;
-    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff149bdf',endColorstr='#ff0480be',GradientType=0);
-    -webkit-box-shadow: inset 0 -1px 0 rgba(0,0,0,.15);
-    -moz-box-shadow: inset 0 -1px 0 rgba(0,0,0,.15);
-    box-shadow: inset 0 -1px 0 rgba(0,0,0,.15);
-    -webkit-box-sizing: border-box;
-    -moz-box-sizing: border-box;
-    box-sizing: border-box;
-    -webkit-transition: width 0.6s ease;
-    -moz-transition: width 0.6s ease;
-    -o-transition: width 0.6s ease;
-    transition: width 0.6s ease;
-}
\ No newline at end of file
+    color: #003A69;
+}
+
+#migrationform #fitem_id_submitbutton, 
+#migration-delete-selected #fitem_id_submitbutton, 
+#fitem_id_enablemigrationtool {
+    width: 50%;
+    margin: 0 auto;
+    background-color: #FFFFFF;
+}
+.fitemtitle {
+    width: 300px !important;
```

Comment:
Could the `!important` here be rendered un-necessary with greater selector specificity?